app/components/roadmap-page/-utils/named-blocks.yaml

```diff
@@ -42,10 +42,20 @@
       </@main>
     {{/x-modal}}
     ```
-  status: in-development
-  statusText: In Implementation
+  updates:
+    - date: June 2018
+      text: |
+        [As mentioned in this comment on the RFC](https://github.com/emberjs/rfcs/pull/226#issuecomment-397419787),
+        named blocks implementation currently needs more design work to work well with angle-bracket components.
+
+        There are apparently some other vague concerns as well.
```

Comment:
What is this :P

Reply:
@locks no idea ... 😀 I'm just the messenger here, it's what was said in the comment. Do you know more details on it?

Reply:
@rwjblue might know.